### PR TITLE
Add SOP for how to store and config a new secret value

### PR DESCRIPTION
**Pull request by ke4**
Changes:
- Add SOP for how to store, config and deploy a new secret value

### CODE REVIEW

File: docs/operations_tasks/add_entry_to_aws_secrets.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+   where `APPLICATION_NAME` reflects the name of the application the secrets will be used
+11. Adds your secret under the environment variables (`spec.template.spec.containers.env`).
+   Define the name and value of your environment variable.
+   If the value comes from AWS secret manager then use this format:
+   ```yaml
+   - name: ENV_VARIABLE_NAME
+     valueFrom:
+       secretKeyRef:
+         key: SERVICE-PASSWORD
+         name: api-keys
```
Comment (aaclan-ebi) [inline docs/operations_tasks/add_entry_to_aws_secrets.md:36]: secret-name

File: docs/operations_tasks/add_entry_to_aws_secrets.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+   ```
+
+## Add secret to the deployment configuration
+12. Modify the `deploy_secrets` shell script under the `scripts` folder and add your secret value below the `get_secret`
+    method call and after the `helm upgrade secrets secrets\` line as you can see it the script.
+   
+## Define your secret in the deployment template
+13. Modify the `deployment.yaml` file under `secrets/templates` folder and add your secret definition under the `data` key.
```
Comment (aaclan-ebi) [inline docs/operations_tasks/add_entry_to_aws_secrets.md:44]: It's either they update an existing secret to have a new key-value or create a new one. 
```suggestion
13. Modify the `deployment.yaml` file under `secrets/templates` folder.
```

File: docs/operations_tasks/add_entry_to_aws_secrets.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+   - select the Services drop-down and in the Security, Identity, & Compliance section click on `Secrets Manager` link,
+   - Type `Secters Manager` into the search bar and click on the appearing `Secrets Manager` link
+4. Click on the Secrets option on the left hand side menu bar
+5. Depending on which environment you would like to add the new secret you should type `ingest/{ENV_NAME}/secrets` to the search bar
+   where `ENV_NAME` is a placeholder and can be the following values: `dev`, `staging` and `prod`.
+6. Click on the name of the secret, for example `ingest/dev/secrets`.
+7. Scroll down to the `Secret value` section and click on the `Retrieve secret value` button.
+8. Click omn the `Edit` button and add your secrets as key/value pairs.
+9. When you finished adding all your new secrets click on the `Save` button to store it.
+
```
Comment (aaclan-ebi) [inline docs/operations_tasks/add_entry_to_aws_secrets.md:24]: could add reference to official AWS documentation https://docs.aws.amazon.com/secretsmanager/latest/userguide/manage_update-secret.html

File: docs/operations_tasks/add_entry_to_aws_secrets.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+3. Go to Secrets Manager. You can do it with any of the follong way:
+   - click `Secters Manager` on the recently visited links section if its is not your first time,
+   - select the Services drop-down and in the Security, Identity, & Compliance section click on `Secrets Manager` link,
+   - Type `Secters Manager` into the search bar and click on the appearing `Secrets Manager` link
+4. Click on the Secrets option on the left hand side menu bar
+5. Depending on which environment you would like to add the new secret you should type `ingest/{ENV_NAME}/secrets` to the search bar
+   where `ENV_NAME` is a placeholder and can be the following values: `dev`, `staging` and `prod`.
+6. Click on the name of the secret, for example `ingest/dev/secrets`.
+7. Scroll down to the `Secret value` section and click on the `Retrieve secret value` button.
+8. Click omn the `Edit` button and add your secrets as key/value pairs.
```
Comment (jacobwindsor) [inline docs/operations_tasks/add_entry_to_aws_secrets.md:22]: on

**Review by aaclan-ebi — APPROVED** [review]

**Review by jacobwindsor — CHANGES_REQUESTED** [review]
Tiny change!

File: docs/operations_tasks/add_entry_to_aws_secrets.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+---
+
+# Add an entry to our secrets that is stored on AWS
+The below steps assume that you have the correct account privileges in AWS.
+
+1. Log in to your HCA's AWS account
+2. Make sure that you are at the right AWS region otherwise you won't able to see/manage our secrets stored in the Secret Manager.
+   The region should be set to: `us-east-1` (N. Virginia)
+3. Go to Secrets Manager. You can do it with any of the follong way:
+   - click `Secters Manager` on the recently visited links section if its is not your first time,
```
Comment (amnonkhen) [inline docs/operations_tasks/add_entry_to_aws_secrets.md:14]:
```suggestion
   - click `Secrets Manager` on the recently visited links section if its is not your first time,
```

File: docs/operations_tasks/add_entry_to_aws_secrets.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: Add an entry to our secrets stored on AWS
+parent: Operations tasks
+---
+
+# Add an entry to our secrets that is stored on AWS
+The below steps assume that you have the correct account privileges in AWS.
+
+1. Log in to your HCA's AWS account
```
Comment (amnonkhen) [inline docs/operations_tasks/add_entry_to_aws_secrets.md:10]:
```suggestion
1. Log in to your [HCA's AWS account](https://embl-ebi.awsapps.com/start#/)
```

Comment (ke4) [reply docs/operations_tasks/add_entry_to_aws_secrets.md:10]: @amnonkhen That link is not working for me.

**Review by jacobwindsor — APPROVED** [review]

**Pull request merged**